packages/pg-v5/commands/promote.js

```diff
@@ -45,6 +45,16 @@ function * run (context, heroku) {
     cli.action.done(cli.color.configVar(backup.name + '_URL'))
   }))
 
+  let db = yield fetcher.addon(app, args.database)
+  let replica = yield heroku.get(`/client/v11/databases/${db.id}`, { host: host(db) })
+
+  if (replica.following) {
+    yield cli.confirmApp(app, flags.confirm, `WARNING:
+      ${cli.color.addon(db.name)} is a follower and not writeable at this time.
```

Comment:
is there a CLI command or devcenter article we could add to this error message so the user knows what to do to move forward?

Reply:
@fivetanley they can type `--confirm` - and I believe the `confirmApp` thing will help that.